Lib/defconQt/fontView.py

```diff
@@ -313,12 +314,13 @@ def __init__(self):
         transformLayout.addWidget(scaleXYBox, l, 5)
         transformGroup.setLayout(transformLayout)
 
-        layerSetGroup = QGroupBox("Layers", self)
+        layerSetGroup = QGroupBox(self.tr("Layers"), self)
         layerSetGroup.setFlat(True)
         layerSetLayout = QVBoxLayout(self)
 
         self.layerSetWidget = QTreeWidget(self)
-        self.layerSetWidget.setHeaderLabels(("Layer Name", "Color"))
+        self.layerSetWidget.setHeaderLabels((self.tr("Layer Name"),
+                                             self.tr("Color")))
```

Comment:
I rather make a new line after function opening brace here, when you make a new line after opening brace you can put just one more level of indent on the next line under PEP8.